src/pathfora.js

```diff
@@ -2871,7 +2871,7 @@
      * @public
      * @description Current version
      */
-    this.version = '0.0.15';
+    this.version = '0.0.16';
```

Comment:
we probably only want to bump the version when we do an update to master from develop, otherwise if multiple branches are going the version number will cause conflicts and weirdness.